### ISSUE TITLE
extract individual species

### ISSUE THREAD

**Issue by DanWismer** [NCC-CNC/landscaperesilience#21]
Would be neat if we could get the unique species that intersect a polygon.

**Comment by edwardsmarc** [comment]
Agreed! This is something I'll need to do for the Canada-wide assessment too. It would make sense for us to share code on that. Basically I want to extract the species intersecting a polygon, and report on the area. Not sure exactly how to do that efficiently yet, maybe exactextractr?

**Comment by ricschuster** [comment]
If you are okay using the 1km grid for this, sparce matrices would be quickest (convert polygon to raster and filter the sparce matric). 
Otherwise I would think exactextractr is the approach to use. Could use the prioritizr functionality for that (https://prioritizr.net/reference/fast_extract.html),

**Comment by edwardsmarc** [comment]
> If you are okay using the 1km grid for this, sparce matrices would be quickest (convert polygon to raster and filter the sparce matric). Otherwise I would think exactextractr is the approach to use. Could use the prioritizr functionality for that (https://prioritizr.net/reference/fast_extract.html),

1km grid would be fine for my use case (ecoregion/ecodistrict level summaries). Not sure if it would work for evaluating small projects in Dan's app though?
